CHANGELOG.md

```diff
@@ -7,6 +7,23 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 Please check our [developers guide](https://gitlab.com/tokend/developers-guide)
 for further information about branching and tagging conventions.
 
+## [Unreleased]
+### Added
+- Displaying total fee in the movement attributes
+- Displaying operation summary in the movement attributes 
+
+### Changed
+- Moved movements module translation filters to a mixin
+- Now displaying only valuable fees in the movement attributes
```

Comment:
After the changes below this row may change

Reply:
Updated